src/components/softwareSkills/SoftwareSkill.css

```diff
@@ -24,6 +22,20 @@
   color: #868e96;
   font-size: 10px;
 }
-.software-skill-inline > i:hover ~ p{
-  color: #645beb;
+.software-skill-inline:hover{
+  transform: scale(0.95);
+  transition: .4s;
+}
+
+.iconify{
```

Comment:
This class name could be changed to something more descriptive or could just use `.software-skill-inline > span`